homeassistant/components/overkiz/number.py

```diff
@@ -94,6 +96,8 @@ async def _async_set_native_value_boost_mode_duration(
         command=OverkizCommand.SET_EXPECTED_NUMBER_OF_SHOWER,
         native_min_value=2,
         native_max_value=4,
+        native_min_key=OverkizState.CORE_MINIMAL_SHOWER_MANUAL_MODE,
```

Comment:
Should we call this `min_value_state` or `default_min_value_state` or something similar? @Tronix117 what do you think?
Let's try to make this as descriptive as possible.